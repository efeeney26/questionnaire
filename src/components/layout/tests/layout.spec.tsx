import React from 'react'
import { shallow } from 'enzyme'

import Layout from '../layout'

describe('Layout', () => {
    it('should render properly with title and children', () => {
        const button = shallow(<Layout title="title">kek</Layout>)
        expect(button).toMatchSnapshot()
    })
})
