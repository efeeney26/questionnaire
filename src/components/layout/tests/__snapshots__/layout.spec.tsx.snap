// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render properly with title and children 1`] = `
<styled.div>
  <styled.h1>
    title
  </styled.h1>
  kek
</styled.div>
`;
